console/jsre/deps/web3.js

```diff
@@ -3848,8 +3848,10 @@ var inputBlockNumberFormatter = function (blockNumber) {
         return undefined;
     } else if (isPredefinedBlockNumber(blockNumber)) {
         return blockNumber;
+    } else if (/^-?\d+$/.test(blockNumber)) {
```

Comment:
Our [docs](https://docs.kaia.io/references/json-rpc/klay/get-code/) says that the second parameter should be one of `IntegerBlockNumber`, `HexadecimalBlockNumber`, `BlockTag` or `BlockHash`. Current code does not allow parameter like `0x1`.

```
klay.getCode("0x98f4aB2c97eFff367827777e2fd96c536f38df1d", "0x1")
Error: input block number(0x1) is invalid
        at inputBlockNumberFormatter (web3.js:3854:11(32))
        at inputDefaultBlockNumberFormatter (web3.js:3843:37(11))
        at web3.js:5169:37(8)
        at map (native)
        at web3.js:5168:35(12)
        at web3.js:5194:34(15)
        at send (web3.js:5219:39(11))
        at <eval>:1:13(4)
```

Reply:
Thank you. Updated to allow hex input (hex number and hash), but does not consider the length validation.